batchflow/models/tf/base.py

```diff
@@ -957,6 +957,11 @@ def get_tensor_config(self, tensor, **kwargs):
         config = {**config, **kwargs}
         return config
 
+    def _set_learning_phase(self, mode):
```

Comment:
Why not `eval`?

Reply:
Why it should be call `eval`, considering this function does not evaluate anything?

Reply:
Like in pytorch - https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module.eval

Reply:
To stay consistent with pytorch we should add two functions: `train` to set learning phase to train mode, `eval` to set learning phase to inference

Reply:
`eval(True)` and `eval(False)` can do the trick